cirq/ops/diagonal_gate.py

```diff
@@ -18,7 +18,7 @@
 passed as a list.
 """
 
-from typing import AbstractSet, Any, Tuple, Iterator, List, Sequence, TYPE_CHECKING, Union
+from typing import AbstractSet, Any, Tuple, Iterator, List, Sequence, TYPE_CHECKING, Union, Optional
```

Comment:
Optional: Should we keep alphabetic order?

Reply:
it wasn't alphabetic?

Reply:
Nope, see "Tuple". Surprised pylint didn't catch it.

Reply:
Indeed. I should be more precise.